server/schedule/operator/builder.go

```diff
@@ -33,6 +33,7 @@ import (
 type ClusterInformer interface {
 	GetBasicCluster() *core.BasicCluster
 	GetOpts() *config.PersistOptions
+	GetConfig() *config.ImmutableConfig
```

Comment:
How about moving it to `Cluster`?

Reply:
this immutable config can be used by some checks or schedulers.

Reply:
fix